src/main/java/argelbargel/jenkins/plugins/gitlab_branch_source/heads/GitLabSCMMergeRequestHead.java

```diff
@@ -32,26 +32,32 @@
 public final class GitLabSCMMergeRequestHead extends GitLabSCMHeadImpl implements ChangeRequestSCMHead {
     private final int id;
     private final String title;
+    private final String description;
     private final GitLabSCMHead sourceBranch;
     private final GitLabSCMBranchHead targetBranch;
     private final boolean mergeable;
     private final boolean merge;
 
 
-    GitLabSCMMergeRequestHead(int id, String title, GitLabSCMHead source, GitLabSCMBranchHead target, boolean mergeable) {
-        this(id, title, source, target, mergeable, false);
+    GitLabSCMMergeRequestHead(int id, String title, String description, GitLabSCMHead source, GitLabSCMBranchHead target, boolean mergeable) {
+        this(id, title, description, source, target, mergeable, false);
     }
 
-    private GitLabSCMMergeRequestHead(int id, String title, GitLabSCMHead source, GitLabSCMBranchHead target, boolean mergeable, boolean merge) {
-        super(source.getProjectId(), title + (merge ? " (merge)" : ""), source.getRevision().getHash(), Messages.GitLabSCMMergeRequest_Pronoun(), MERGE_REQUESTS);
+    private GitLabSCMMergeRequestHead(int id, String title, String description, GitLabSCMHead source, GitLabSCMBranchHead target, boolean mergeable, boolean merge) {
+        super(source.getProjectId(), title + (merge ? "-merged" : ""), source.getRevision().getHash(), Messages.GitLabSCMMergeRequest_Pronoun(), MERGE_REQUESTS);
         this.id = id;
         this.title = title;
+        this.description = description;
         this.sourceBranch = source;
         this.targetBranch = target;
         this.mergeable = mergeable;
         this.merge = merge;
     }
 
+    public String getDescription() {
```

Comment:
can not figure out where this function is used

Reply:
Did not look in the code yet, but i think it is (was?) used as the job's description in Jenkins below the job name

Reply:
Its used in `SourceActions.java` on line 78: 
```
actions.add(createHeadMetadataAction(((GitLabSCMMergeRequestHead) head).getDescription(), ((GitLabSCMMergeRequestHead) head).getSource(), null, linkAction.getUrlName()));
```